### PR TITLE
Add `extism_version` to Host SDK and all clients

### PR DESCRIPTION
**Pull request by bhelx**
Adds a new Host SDK call `extism_version` which returns the version string.

### CODE REVIEW

File: runtime/src/sdk.rs [hunk 1]
```diff
@@ -385,3 +385,8 @@ pub unsafe extern "C" fn extism_log_file(
     }
     true
 }
+
+#[no_mangle]
+pub unsafe extern "C" fn extism_version() -> *const u8 {
+    env!("CARGO_PKG_VERSION").as_ptr()
```
Comment (bhelx) [inline runtime/src/sdk.rs:391]: Is this the right manifest to grab the version from? it seems we are on release `v0.0.1-rc.4` but this one is showing `rc.3`.

Comment (bhelx) [reply runtime/src/sdk.rs:391]: My mistake. We identified my main branch was pointed to the wrong origin.

**Comment by zshipko** [comment]
Good idea! it looks like Go and Haskell need to be updated too

File: runtime/extism.h [hunk 2]
```diff
@@ -106,3 +106,5 @@ const uint8_t *extism_plugin_output_data(struct ExtismContext *ctx, ExtismPlugin
  * Set log file and level
  */
 bool extism_log_file(const char *filename, const char *log_level);
+
+const uint8_t *extism_version(void);
```
Comment (bhelx) [inline runtime/extism.h]: This appears to be generated but i'm not sure from where. missing a docstring

Comment (zshipko) [reply runtime/extism.h]: I think converting this comment to a doc comment should copy the documentation over: https://github.com/extism/extism/pull/30/files#diff-9c18909a3b1c4dcdf57dc4edc707e98ba17728203f7c17d573a5d1de60ba9095R389

Comment (zshipko) [reply runtime/extism.h]: (It's generated from sdk.rs using cbindgen in the build script)

Comment (bhelx) [reply runtime/extism.h]: yes, appears i just annotated that comment incorrectly

**Review by zshipko — APPROVED** [review]

**Pull request merged**